backend/ticketvise/media/tickets/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Wat doet deze gitignore?

Reply:
Dat zorgt ervoor dat de bestanden die hier worden geupload (attachments) niet worden mee gepushed in een commit.

Reply:
Ah, ik dacht dat de gehele media folder al werd genegeerd.

Reply:
Volgens mij niet, want ik zag dat wanneer ik met attachments bezig was dat er dingen gecommit werden, dus dit moet er zeker van zijn dat er geen media bestanden meer meekomen :)